libraries/botbuilder-storage-dynamodb/README.md

```diff
@@ -0,0 +1,18 @@
+# Bot Builder DynamoDB Storage
+
+This is a simple storage adapter for storing BotState in DynamoDB. To use:
+
+```js
+import { DynamoDBStorage } from '@botbuildercommunity/storage-dynamodb';
+
+const dynamoDBStorage = new DynamoDBStorage('table-name', 'us-east-1');
```

Comment:
No credentials?

Reply:
Typically credentials are set as env vars, and `aws-sdk` picks them up. That said, it would be nice to add the ability to provide explicit creds. Will do.

Reply:
You could be correct. One of the middleware modules read directly from the `.env` file. If that's how this handles it, please add those instructions to the `README.md` file.

Reply:
I've added the ability to specify explicit creds. Will update the README on how to use other AWS credentials mechanisms.

Reply:
The update to the README is complete.